lab_1/main.py

```diff
@@ -125,4 +208,15 @@ def sort_concordance(tokens: list, word: str, left_context_size: int, right_cont
     left_sort = True
     --> [['dog', 'is', 'happy', 'but', 'the', 'cat'], ['man', 'is', 'happy', 'the', 'dog', 'is']]
     """
-    pass
+    if isinstance(tokens, list) and isinstance(word, str) and isinstance(left_context_size, int)\
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.